recipes/libvips/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+export LD_LIBRARY_PATH="${PREFIX}/lib:${LD_LIBRARY_PATH}"
+export CFLAGS="-O3 ${CFLAGS}"
+export CXXFLAGS="-O3 ${CXXFLAGS}"
+
+./configure --prefix="${PREFIX}"
+( make > make.out ) || ( cat make.out && exit 1 )
```

Comment:
make.out won't really be stored anywhere permanent, maybe just replace with this `make`